.gitignore

```diff
@@ -78,8 +78,11 @@ documentation/plan_functions.md
 documentation/bolt_command_reference.md
 documentation/bolt_configuration_reference.md
 documentation/privilege_escalation.md
-documentation/bolt_defaults_reference.md
+documentation/bolt_pwsh_reference.md
+documentation/bolt_project_reference.md
```

Comment:
This looks like a dup of the line below it.

Reply:
good catch